modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js

```diff
@@ -313,6 +324,9 @@ export const getUserAccount = gql`
 					name
 				}
 			}
+			roleBriefs {
```

Comment:
roleBriefs should be placed in alphabetical order, below name

Reply:
Ok, thanks